server.js

```diff
@@ -0,0 +1,7 @@
+var express = require('express')
+var app = express()
+app.get('/ping', (req, res) =>{
+    res.send('Hello World')
+})
+
+app.listen(3000);
```

Comment:
It's a good practice to make the listening port configurable. You can use an environment variable with a fallback to a default port. [medium]